.github/workflows/build-android.yml

```diff
@@ -0,0 +1,60 @@
+name: Build Rust
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}-build-android"
+  cancel-in-progress: true
+
+on:
+  workflow_call:
+
+env:
+  CARGO_TERM_COLOR: always
+  CARGO_NET_GIT_FETCH_WITH_CLI: true
+  CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
+
+jobs:
+  build-android:
+    if: github.repository == 'wireapp/core-crypto'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: "17"
+          distribution: "adopt"
+      - name: Gradle Setup
+        uses: gradle/actions/setup-gradle@v4
+      - name: Validate Gradle wrapper
+        uses: gradle/actions/wrapper-validation@v4
+      - name: "Setup rust"
+        uses: actions-rust-lang/setup-rust-toolchain@v1  # this implicitly caches Rust tools and build artifacts
+        with:
+          rustflags: ''
+          target: "armv7-linux-androideabi,aarch64-linux-android,x86_64-linux-android"
+      - name: Setup cargo-make
+        uses: davidB/rust-cargo-make@v1
+      - name: Setup Android SDK
+        uses: android-actions/setup-android@v3
+      - name: Install NDK 25
+        run: echo "y" | sdkmanager --install "ndk;25.2.9519653"
+      - name: Build Android package
+        env:
+          ANDROID_NDK_HOME: /usr/local/lib/android/sdk/ndk/25.2.9519653
+        run: |
+          cd crypto-ffi
+          cargo make android
+      - name: Build package
+        run: |
+          cd crypto-ffi/bindings
+          ./gradlew android:build -x lint -x lintRelease
+      - uses: actions/upload-artifact@v4
+        with:
+          name: android-target-${{ github.run_id }}
+          path: target/*android*
+          retention-days: 1
+      - uses: actions/upload-artifact@v4
+        with:
+          name: android-${{ github.run_id }}
+          path: crypto-ffi/bindings
```

Comment:
Why not upload `crypto-ffi/bindings/android`?

Reply:
I ~~couldn't~~ didn't want to take the time to prove that it didn't require other components, like `uniffi-android`. Total size of this artifact is ~8Mb, which should be fine, I think.

Reply:
Oof, I didn't see `uniffi-android`, good point. :+1: